corehq/apps/app_manager/static/app_manager/js/detail-screen-config.js

```diff
@@ -415,6 +418,11 @@ hqDefine('app_manager/js/detail-screen-config.js', function () {
             self.add_item('responses');
         }
 
+        self.lookup_attach_results = ko.observable(state.lookup_display_results);  // We don't store lookup_attach_results
```

Comment:
Yeah, I think the "Attach results to case list" is redundant because we always want to filter the case list by the callout result (I think). Having just the "Display results in case list" should suffice.